testing/features/support/components/pagination/example.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Pagination
+  class Example < ::Base
+    set_url "/iframe.html?id=components-pagination--example&args=&viewMode=story"
+
+    element :current_page, ".cads-paging__current a"
```

Comment:
A better selector for this might be `[aria-current="page"]` as you would then be:
 - not using a styling class in a test (this makes the test fragile)
 - also testing accessibility 

If you change the selector to this, then you can just do `expect(@component).to have_current_page` in the step definitions.